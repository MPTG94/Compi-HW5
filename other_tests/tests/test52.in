void maxTestsTestContinue() {
	int x;
	while(true) {
		if (x == 10) {
			break;
		}
		x = x + 1;
		if (x / 2 == x - x / 2) {
			continue;
		}
		printi(x);
		print("x is odd");
	}
}

void main() {
     maxTestsTestContinue();
}
