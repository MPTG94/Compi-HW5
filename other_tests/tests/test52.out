1
x is odd
3
x is odd
5
x is odd
7
x is odd
9
x is odd
